### PR TITLE
VTCCA-4506 extracted header to new file

### PR DESCRIPTION
**Pull request by DanSmith98**
https://jira.tools.tax.service.gov.uk/browse/VTCCA-4506

### CODE REVIEW

**Comment by habeeb01** [comment]
LGTM 👍

File: app/views/includes/assessmentHeading.scala.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+@import utils.Formatters
+
+@(localAuthorityReference: String, address: String)(implicit messages: Messages)
+
+
```
[the remark this answers is not in the document]
Comment (darrell-plant) [reply app/views/includes/assessmentHeading.scala.html:5]: I think you have done that 👍

**Comment by darrell-plant** [comment]
LGTDz 👍

File: app/views/includes/assessmentHeading.scala.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (DanSmith98) [reply app/views/includes/assessmentHeading.scala.html:5]: The code is very similar, and the header is visually the same as the other services I looked at. Only difference is some of them have both authority codes - like 01234/5678910 - but the first data isn't and wasn't already present on any PL pages.

**Pull request merged**